examples/deployment/deployment.tf

```diff
@@ -21,19 +21,11 @@ resource "ec_deployment" "example_minimal" {
   version                = "7.9.2"
   deployment_template_id = "aws-io-optimized-v2"
 
-  elasticsearch {}
-
-  kibana {}
-
-  apm {
-    topology {
-      size = "0.5g"
+  elasticsearch {
+    config {
+	user_settings_yaml = file("./es_settings.yaml")
     }
   }
 
-  enterprise_search {
-    topology {
-      zone_count = 1
-    }
-  }
-}
```

Comment:
The linter is failing on this file :( It might be because you added a newline at the end of the file?

Reply:
sigh that linter! i'll fix.